From ded5104ad5f6c31286c6c7f44ead1983ee35cb03 Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Fri, 12 Mar 2021 06:45:43 +0100
Subject: [PATCH] systemd-halt.service: always poweroff on halt

halt without poweroff makes no sense on our hipos hardware.

Signed-off-by: Steffen Sledz <sledz@dresearch-fe.de>
---
 units/systemd-halt.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-halt.service b/units/systemd-halt.service
index cd16d1de20..3f96b9b30d 100644
--- a/units/systemd-halt.service
+++ b/units/systemd-halt.service
@@ -16,4 +16,4 @@ After=shutdown.target umount.target final.target
 
 [Service]
 Type=oneshot
-ExecStart=systemctl --force halt
+ExecStart=systemctl --force reboot
-- 
2.26.2

